### ISSUE TITLE
indexeddb wrapper fails occasionally on startup

### ISSUE THREAD

**Issue by xnopasaranx** [viur-framework/viur-vi#55]
When I use the pyodide vi locally, I sometimes get a strange error when logging in to the vi for the first time after starting up:
```Traceback (most recent call last):
pyodide.asm.js:8   File "/lib/python3.7/site-packages/vi/network.py", line 67, in onReadyStateChange
pyodide.asm.js:8     self.cb.onCompletion( self.req.responseText )
pyodide.asm.js:8   File "/lib/python3.7/site-packages/vi/network.py", line 338, in onCompletion
pyodide.asm.js:8     s( self )
pyodide.asm.js:8   File "/lib/python3.7/site-packages/vi/admin.py", line 107, in getCurrentUserSuccess
pyodide.asm.js:8     self.startup()
pyodide.asm.js:8   File "/lib/python3.7/site-packages/vi/admin.py", line 126, in startup
pyodide.asm.js:8     self.topBar.invoke()
pyodide.asm.js:8   File "/lib/python3.7/site-packages/vi/widgets/topbar.py", line 70, in invoke
pyodide.asm.js:8     conf["mainWindow"].logWdg = widget()
pyodide.asm.js:8   File "/lib/python3.7/site-packages/vi/log.py", line 121, in __init__
pyodide.asm.js:8     idb.dbAction( "createStore", "vi_log", None, { "autoIncrement": True } )
pyodide.asm.js:8   File "/lib/python3.7/site-packages/vi/utils.py", line 244, in dbAction
pyodide.asm.js:8     self.dbVersion +=1
pyodide.asm.js:8 TypeError: unsupported operand type(s) for +=: 'NoneType' and 'int'
pyodide.asm.js:8 
pyodide.asm.js:8 Uncaught Error: Traceback (most recent call last):
  File "/lib/python3.7/site-packages/vi/network.py", line 67, in onReadyStateChange
    self.cb.onCompletion( self.req.responseText )
  File "/lib/python3.7/site-packages/vi/network.py", line 338, in onCompletion
    s( self )
  File "/lib/python3.7/site-packages/vi/admin.py", line 107, in getCurrentUserSuccess
    self.startup()
  File "/lib/python3.7/site-packages/vi/admin.py", line 126, in startup
    self.topBar.invoke()
  File "/lib/python3.7/site-packages/vi/widgets/topbar.py", line 70, in invoke
    conf["mainWindow"].logWdg = widget()
  File "/lib/python3.7/site-packages/vi/log.py", line 121, in __init__
    idb.dbAction( "createStore", "vi_log", None, { "autoIncrement": True } )
  File "/lib/python3.7/site-packages/vi/utils.py", line 244, in dbAction
    self.dbVersion +=1
TypeError: unsupported operand type(s) for +=: 'NoneType' and 'int'

    at _hiwire_throw_error (pyodide.asm.js:8)
    at wasm-function[358]:0xfc6c0
    at wasm-function[356]:0xfbff0
    at Module.__pyproxy_apply (pyodide.asm.js:8)
    at Object.apply (pyodide.asm.js:8)```

**Comment by phorward** [comment]
Hello, I had this too yesterday, and fixed temporarily disabled it with #f4f99faaa46ae80c46a5c7f1eca008e9bbd2d2a6. This feature was written by @XeoN-GHMB so he's in position to fix it.

**Comment by akelch** [comment]
in case of an invalid cache version it is now initialized with 0
